packages/toolchain/dasel/build.yaml

```diff
@@ -0,0 +1,20 @@
+requires:
+  - name: "golang"
+    category: "build"
+    version: ">=0"
+env:
+  - PATH=$PATH:/usr/local/go/bin
+  - GOPATH=/luetbuild/go
+  - GO111MODULE=on
```

Comment:
no fips?

Reply:
nope no actual need, would be actually just used by the CI